docs/shared/core-features/automate-updating-dependencies.md

```diff
@@ -66,6 +66,28 @@ After you run all the migrations, you can remove `migrations.json` and commit an
 
 Note: You may want to keep the `migrations.json` until every branch that was created before the migration has been merged. Leaving the `migrations.json` in place allows devs to run `nx migrate --run-migrations` to apply the same migration process to their newly merged code as well.
 
+## Keep Nx Packages on the Same Version
```

Comment:
should we add a link to this page inside the call outs added to each plugin page?

Reply:
or maybe at least some info about 'run nx report to see what versions are out of sync easily'.

Reply:
Good idea.